charts/posthog/ALL_VALUES.md

```diff
@@ -114,6 +114,7 @@ The following table lists the configurable parameters of the PostHog chart and t
 | ingress.nginx.redirectToTLS | bool | `true` | Whether to redirect to TLS with nginx ingress. |
 | ingress.annotations | object | `{}` | Extra annotations |
 | ingress.secretName | string | `nil` | TLS secret to be used by the ingress. |
+| ingress.class | string | `nil` | A custom ingress class |
```

Comment:
Can we say that this "Specifies the ingressClassName used. Defaults to `nginx` if `ingress.type` is `nginx` or `gcp` if `ingress.type` is `clb`"

Feels a bit messy, @guidoiaquinti any thoughts?